data/human/human missions.txt

```diff
@@ -5984,6 +5984,7 @@ mission "Argosy Hijacking"
 		on kill
 			set "destroyed hijacked argosy"
 			dialog `You have destroyed the Moonshake Express. The ship's captain won't be impressed.`
+			fail
```

Comment:
```suggestion
```
From the wiki:

`save: The mission fails if the given NPC is destroyed.`

I strongly recommend playtesting any and all mission changes to see if the mission is actually failing or not/if the current syntax already is correct.

Reply:
Forgot about the save. You're correct.